.github/workflows/build-test.yml

```diff
@@ -44,7 +44,7 @@ jobs:
             fail-fast: false
             matrix:
                 runConfig: [ {cloud: 'AWS', javaVersion: '8'}, {cloud: 'GCP', javaVersion: '11'}, {cloud: 'AZURE', javaVersion: '17'}, {cloud: 'AWS', javaVersion: '21'}]
-                category: ['TestCategoryResultSet,TestCategoryOthers,TestCategoryLoader', 'TestCategoryConnection,TestCategoryStatement', 'TestCategoryArrow,TestCategoryCore', 'TestCategoryFips']
+                category: ['TestCategoryResultSet,TestCategoryOthers,TestCategoryLoader', 'TestCategoryConnection,TestCategoryStatement', 'TestCategoryArrow,TestCategoryCore', 'TestCategoryDiagnostic', 'TestCategoryFips']
```

Comment:
let's add `TestCategoryDiagnostic` to the first group `TestCategoryResultSet,TestCategoryOthers,TestCategoryLoader`

Reply:
Done